src/http/request_parser.rl

```diff
@@ -76,14 +76,13 @@ action checkpoint {
 }
 
 action assign_field {
```

Comment:
The very same code exists in response_parser.rl which also deserves the optimization

Reply:
done